spec/regression/access-restrictions/model/file.graphqls

```diff
@@ -0,0 +1,4 @@
+type File @rootEntity(flexSearch: true, permissionProfile: "restricted") {
+    name: String @flexSearch
+    accessGroup: String @flexSearch @accessField
```

Comment:
it is not necessary to put @accessField on the accessGroup field if you use the accessGroup field in the old way (`"restrictToAccessGroups": [ "foo_$1" ]`) right?
Otherwise that would be a huge breaking change.

Reply:
yes, I just called it accessGroup here because I was lazy and did not want to change the tests. The existing tests based don accessGroup still pass, so they don't require the annotation.